llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -333,6 +333,16 @@ unsigned SubtargetEmitter::cpuNames(raw_ostream &OS) {
   return Names.size();
 }
 
+static void checkDuplicateCPUFeatures(StringRef CPUName, StringRef FeatureKind,
+                                      ConstRecVec Features) {
```

Comment:
nit: use `ArrayRef<const Record *>` instead of `ConstRecVec`.?

Reply:
Oops, for some reason I mistakenly thought `ConstRecVec` is a typedef to a std::vector reference. It's fixed now.